Mathlib/Analysis/Normed/Lp/ProdLp.lean

```diff
@@ -563,6 +592,22 @@ theorem prod_infty_equiv_isometry [PseudoEMetricSpace α] [PseudoEMetricSpace β
       simpa only [ENNReal.div_top, ENNReal.toReal_zero, NNReal.rpow_zero, ENNReal.coe_one,
         one_mul] using prod_antilipschitzWith_equiv ∞ α β x y)
 
+theorem prod_lipschitzWith_ofLp [PseudoEMetricSpace α] [PseudoEMetricSpace β] :
+    LipschitzWith 1 (@WithLp.ofLp p (α × β)) :=
+  prod_lipschitzWith_equiv_aux p α β
+
+theorem prod_antilipschitzWith_ofLp [PseudoEMetricSpace α] [PseudoEMetricSpace β] :
+    AntilipschitzWith ((2 : ℝ≥0) ^ (1 / p).toReal) (@WithLp.ofLp p (α × β)) :=
+  prod_antilipschitzWith_equiv_aux p α β
+
+theorem prod_infty_ofLp_isometry [PseudoEMetricSpace α] [PseudoEMetricSpace β] :
```

Comment:
```suggestion
theorem isometry_prod_infty_ofLp [PseudoEMetricSpace α] [PseudoEMetricSpace β] :
```

Reply:
Also changed the original theorem name to `isometry_prod_infty_equiv` for consistency.

Reply:
Oh I was thinking you could leave that one alone since you're about to deprecate it anyway